README.md

```diff
@@ -2,52 +2,43 @@
+This repository contains the functionality to standardize the data of [Boets et al. (2016)](http://www.aquaticinvasions.net/2016/AI_2016_Boets_etal.pdf) to both a [Darwin Core checklist](https://www.gbif.org/dataset-classes) and [Darwin Core occurrence dataset](https://www.gbif.org/dataset-classes) that can be harvested by [GBIF](http://www.gbif.org). The repository also contains the code for a [website](http://trias-project.github.io/alien-macroinvertebrates/map.html) to explore the occurrence data. It was developed for the [TrIAS project](http://trias-project.be).
 
-## Results
+## Workflow
 
-### Occurrence dataset
+* Checklist dataset: [source data](https://github.com/trias-project/alien-macroinvertebrates/blob/master/data/raw/AI_2016_Boets_etal_Supplement.xls) → Darwin Core [mapping script](http://trias-project.github.io/alien-macroinvertebrates/dwc_checklist.html) → generated [Darwin Core files](https://github.com/trias-project/alien-macroinvertebrates/blob/master/data/processed/dwc_checklist)
+* Occurrence dataset: [source data](https://github.com/trias-project/alien-macroinvertebrates/blob/master/data/raw/denormalized_observations.csv) → Darwin Core [mapping script](http://trias-project.github.io/alien-macroinvertebrates/dwc_occurrence.html) → generated [Darwin Core files](https://github.com/trias-project/alien-macroinvertebrates/tree/master/data/processed/dwc_occurrence)
 
-* Description of the [Darwin Core mapping](src/dwc_occurrence.md) (= a rendition of the [mapping script](src/dwc_occurrence.R))
-* Generated [Darwin Core files](data/processed/dwc_occurrence/)
-* Published [dataset on the IPT](https://ipt.inbo.be/resource?r=alien-macroinvertebrate-occurrences)
-* Published [dataset on GBIF](https://doi.org/10.15468/xjtfoo)
-* Article for this dataset:
+## Published dataset
 
-> Boets P, Brosens D, Lock K, Adriaens T, Aelterman B, Mertens J & Goethals PLM (2016) Alien macroinvertebrates in Flanders (Belgium). Aquatic Invasions 11: 131-144. <https://doi.org/10.3391/ai.2016.11.2.03> <http://www.aquaticinvasions.net/2016/AI_2016_Boets_etal.pdf>
+* [Checklist dataset on the IPT](https://ipt.inbo.be/resource?r=alien-macroinvertebrates-checklist)
+* [Checklist dataset on GBIF](https://doi.org/10.15468/yxcq07)
 
-### Checklist dataset
+---
 
-* Description of the [Darwin Core mapping](src/dwc_checklist.md) (= a rendition of the [mapping script](src/dwc_checklist.R))
-* Generated [Darwin Core files](data/processed/dwc_checklist/)
-* Published [dataset on the IPT](https://ipt.inbo.be/resource?r=alien-macroinvertebrates-checklist)
```

Comment:
Just thinking out loud, but should we mention the publication of the dataset on the IPT here? It appears to me that this is just an intermediate step for the publication on GBIF (which is the one people will be visiting).

Reply:
Agree, but it is a very useful link to us.